tcms/testruns/static/testruns/js/get.js

```diff
@@ -9,6 +9,9 @@ const allExecutionStatuses = {},
     autocomplete_cache = {}
 
 $(document).ready(() => {
+    $(function () {
+        $('[data-toggle="tooltip"]').tooltip()
+    })
```

Comment:
Let's remove this block. Regular tooltips are fine. 

If you insist on keeping the nicer tooltips the onclick handler of the buttons should hide them as well. Try these steps:
1) hover the mouse on the Stop button => tooltip appears
2) click the Stop button without moving the mouse => timestamp is updated; button hides itself

however the tooltip stays on the screen.

Reply:
@atodorov basically I tried and on to reproduce this problem and it didn't really appear in my case (MacBook pro Chrome and Safari), but I have no hard feeling about it. I could delete it if you think it's not needed

Reply:
I updated this function to explicitly call the trigger the tooltip when the mouse hovers only, this was one of the solutions on stack overflow, but not sure if it has an effect on your side.